lib/pivo_flow/pivotal.rb

```diff
@@ -252,12 +252,17 @@ def deliver_story story_id
     end
 
     def remove_story_id_file
-      FileUtils.remove_file(@story_id_file_path)
+      FileUtils.remove_file(current_story_id_file_path)
     end
 
     def save_story_id_to_file story_id
-      FileUtils.mkdir_p(@story_id_tmp_path)
-      File.open(@story_id_file_path, 'w') { |f| f.write(story_id) }
+      ensure_tmp_directory_exists
```

Comment:
separating this method doesn't give us a lot - it's one line only - what do you think?

Reply:
I'm personally a fan of method names that express intent so that you can quickly read what's going on without having to parse out the details of the code. Your repo though so I'm good with either way. I'd just lean towards keeping it as it reads better.

Reply:
I get it, but in this case, let's not add another method, please.